app/client/src/components/shared/CharacteristicsSelect.tsx

```diff
@@ -44,41 +60,98 @@ export function CharacteristicsSelect({
 
   return (
     <div css={selectContainerStyles}>
-      <label css={selectLabelStyles} htmlFor={inputId}>
-        {label}
-      </label>
+      <span css={selectLabelStyles}>
+        Filter by{' '}
+        <GlossaryTerm term="Characteristics">Characteristics</GlossaryTerm>:
+      </span>
```

Comment:
I think the associated label is preferred over just an `aria-label`. In Firefox, a warning is generated here that says: "each form element should have a visible label". While it's visually obvious the text is for the label, I think minimizing these warnings makes it easier to review the site's accessibility in the future. Also, the screen reader reads the text with the glossary label fine (it just reads the inner text).
```suggestion
      <label css={selectLabelStyles} htmlFor={inputId}>
        Filter by{' '}
        <GlossaryTerm term="Characteristics">Characteristics</GlossaryTerm>:
      </label>
```

Reply:
The reason I moved away from the label is because it makes it so that clicking on the glossary term does not open the glossary. Instead of opening the glossary, it focuses on the associated input. 

I followed what we did on the state advanced search pages, which passes wave and axe.

I can look into other potential workarounds to get rid of the warning in Firefox. I suppose one workaround would be to pull the glossary term out of the label.

Reply:
That's right, I suppose I tested the keyboard after reverting but didn't try clicking. In the case of pulling out the glossary term, it probably is better to keep the `aria-label` there.

Reply:
Yeah that would make the label not make sense. I'll stick with the `aria-label`.